joern-cli/frontends/solidity2cpg/src/main/scala/io/joern/solidity2cpg/passes/AstCreator.scala

```diff
@@ -910,29 +859,30 @@ class AstCreator(filename: String, sourceUnit: SourceUnit, global: Global) exten
     }
     else
       operation.operator match {
+        // TODO Can `NOT` really be postfix?
```

Comment:
From [TutorialsPoint](https://www.tutorialspoint.com/solidity/solidity_operators.htm) it doesn't seem like it lol. @MatthewEthanTam can you confirm this? e.g. is `x!` a valid expression versus prefix `!x`.